lib/rubocop/cached_data.rb

```diff
@@ -48,10 +48,18 @@ def deserialize_offenses(offenses)
       source_buffer = Parser::Source::Buffer.new(@filename)
       source_buffer.source = File.read(@filename, encoding: Encoding::UTF_8)
       offenses.map! do |o|
-        location = Parser::Source::Range.new(source_buffer,
-                                             o['location']['begin_pos'],
-                                             o['location']['end_pos'])
-        Cop::Offense.new(o['severity'], location, o['message'], o['cop_name'], o['status'].to_sym)
+        range = offense_from_cache(o, source_buffer)
+        Cop::Offense.new(o['severity'], range, o['message'], o['cop_name'], o['status'].to_sym)
```

Comment:
Can you rename to the following?

```suggestion
        location = location_from_source_buffer(o, source_buffer)
        Cop::Offense.new(o['severity'], location, o['message'], o['cop_name'], o['status'].to_sym)
```